app/Contracts/AdminSettingsContract.php

```diff
@@ -0,0 +1,13 @@
+<?php
+namespace App\Contracts;
+
+use App\Models\OrganizationSetting;
+
+interface AdminSettingsContract
+{
+    public function currentOrganizationSettings($organizationId);
```

Comment:
Function name is not descriptive to caller, should name it something like getOrganizationSettings.